open-source-services.md

```diff
@@ -14,7 +14,7 @@ This represents what we expect from any _service_ claiming to be Open Source.
 These requirements do not represent best practices, but are the bare minimum for Open Source development of services.
 Most Open Source models (beyond “throw it over the wall” behavior) are covered by these requirements in one way or another:
 
- 1. **All the service's code and assets necessary to operate the service are shared under an open source license, and publicly accessible.**
+ 1. **All the service's code and assets necessary to operate the service are shared under an Open Source license, and publicly accessible.**
```

Comment:
Makes sense.

Reply:
Yeah, consistency, and I've moved to using Open whenever it's the case.